src/routes/conversation.js

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express'
+import {
+  createConversation,
+  findAllConversationsByUserId
+} from '../controllers/conversation.js'
+import { validateAuthentication } from '../middleware/auth.js'
+
+const router = Router()
+
+router.post('/', validateAuthentication, createConversation)
+router.get(
+  '/user/:userId',
```

Comment:
Conventionally, I expect this path to be `/users/:userId/conversations` - so move this routing to the users router.

Keep the controller functions in the conversation controller,

Reply:
Resolved